models/_sector/contracts/zora/contracts_zora_base_iterated_creators.sql

```diff
@@ -0,0 +1,15 @@
+ {{
+  config(
+        schema = 'contracts_zora',
+        alias = 'base_iterated_creators',
+        materialized ='incremental',
+        file_format ='delta',
+        incremental_strategy='merge',
+        unique_key = ['blockchain', 'contract_address'],
```

Comment:
unique keys don't match unique combo of columns test in schema file. plz update one or the other to get them to match and assign the proper keys

Reply:
this applies to all zora spells in here

Reply:
modding schema - created_tx_hash is leftover from legacy versions